CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+## I want to contribute to Wire
+
+You can contribute to Wire in several ways:
+
+## Finding bugs
+
+If you find a bug in how Wire apps work, please submit a ticket to [our support](https://support.wire.com) and we will keep you informed about the progress.
+Alternatively, you can submit your finding on [wire issues](https://github.com/wireapp/wire/issues). Please make sure to provide as much information as possible to help us reproduce the issue.
+
+## Contributing to the code
+
+If you wish to contribute source code to one of our repositories you have to sign
+our [Contributor Agreement](https://github.com/wireapp/wire/raw/master/assets/Wire%20Contributor%20Agreement.pdf)
+first.
+
+When you submit your first pull request, you can sign the agreement electronically by filling in the
+required information. You will not have to sign it again for subsequent pull requests from the same
+GitHub account.
+
+When opening a pull request, please make sure to follow the next guidelines:
+
+- Maker sure to fill in the pull request template to the fullest extent possible, this will help us
+  understand faster the changes proposed.
+- Maker sure to run the tests and linters before submitting the pull request.
```

Comment:
```suggestion
- Make sure to fill in the pull request template to the fullest extent possible, this will help us
  understand faster the changes proposed.
- Make sure to run the tests and linters before submitting the pull request.
```